crates/cli/commands/src/p2p/rlpx.rs

```diff
@@ -0,0 +1,52 @@
+//! RLPx subcommand of P2P Debugging tool.
+
+use clap::{Parser, Subcommand};
+use enr::Enr;
+use reth_ecies::stream::ECIESStream;
+use reth_eth_wire::{HelloMessage, UnauthedP2PStream};
+use reth_network::config::rng_secret_key;
+use reth_network_peers::{pk2id, NodeRecord};
+use secp256k1::SECP256K1;
+use tokio::net::TcpStream;
+
+/// RLPx commands
+#[derive(Parser, Debug)]
+pub struct Command {
+    #[clap(subcommand)]
+    subcommand: Subcommands,
+}
+
+impl Command {
+    // Execute `p2p rlpx` command.
+    pub async fn execute(self) -> eyre::Result<()> {
+        match self.subcommand {
+            Subcommands::Ping { node } => {
+                let key = rng_secret_key();
+                let enr = node.parse::<Enr<secp256k1::SecretKey>>().unwrap();
```

Comment:
would be more helpful if this parsed an enode into an enr, rather than the serialised base 64 enr string, could open an issue for addressing this separately though